bundles/org.openhab.binding.squeezebox/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -7,7 +7,7 @@
 	<bridge-type id="squeezeboxserver">
 		<label>SqueezeBox Server</label>
 		<description>This is a SqueezeBox Server instance.</description>
-
+		<semantic-equipment-tag>AudioVisual</semantic-equipment-tag>
```

Comment:
Similar to the UniFi Controller, this is a piece of software used as bridge for the player Things. I think `AudioVisual` is probably wrong, but I can't propose anything else.

Reply:
We have a `MediaPlayer` (which it is not); and the obvious equipment tag is `MediaServer` (which it is; but which tag is missing). But in any case it is certainly "something A/V" ..

Reply:
> I can't propose anything else.

So I shall keep `AudioVisual` ..